src/System.Private.ServiceModel/src/System/ServiceModel/Description/CustomAttributeProvider.cs

```diff
@@ -77,13 +77,17 @@ public object[] GetCustomAttributes(Type attributeType, bool inherit)
             switch (this.ProviderType)
             {
                 case AttributeProviderType.Type:
-                    return this.Type.GetTypeInfo().GetCustomAttributes(attributeType, inherit).ToArray();
+                    //return this.Type.GetTypeInfo().GetCustomAttributes(attributeType, inherit).ToArray();
+                    return this.Type.GetTypeInfo().GetCustomAttributes().ToArray();
```

Comment:
Thanks for going after the root cause @StephenBonikowsky .  This PR will need a couple tweaks before merging.

We should not modify CustomAttributeProvider like this because that changes its general behavior for our specific failure case.  I believe there is an alternate form of GetCustomAttributes which does not take an 'attributeType' parameter, and we should be calling that one instead.